metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery.py

```diff
@@ -631,20 +632,27 @@ def _process_project(
                 )
 
             self.report.set_project_state(project_id, "Usage Extraction")
-            yield from self.generate_usage_statistics(project_id)
+            yield from self.generate_usage_statistics(
+                project_id, db_tables=db_tables, db_views=db_views
+            )
 
         if self.config.profiling.enabled:
             logger.info(f"Starting profiling project {project_id}")
             self.report.set_project_state(project_id, "Profiling")
             yield from self.profiler.get_workunits(
                 project_id=project_id,
-                tables=self.db_tables,
+                tables=db_tables,
             )
 
-    def generate_lineage(self, project_id: str) -> Iterable[MetadataWorkUnit]:
+    def generate_lineage(
+        self,
+        project_id: str,
+        db_tables: Dict[str, List[BigqueryTable]],
+        db_views: Dict[str, List[BigqueryView]],
+    ) -> Iterable[MetadataWorkUnit]:
         logger.info(f"Generate lineage for {project_id}")
-        for dataset in self.db_tables:
-            for table in self.db_tables[dataset]:
+        for dataset in db_tables:
```

Comment:
Why not use .keys() here too? Like we do below. Whichever way we go let's keep consistency

Reply:
Fixed